scripts/packaging/webots_distro.py

```diff
@@ -53,6 +53,7 @@
 
 # sanity check: the following command will display an error if webots fails to start
 status = os.system(f'bash -c "{webots_command} --sysinfo"')
+print(status)
```

Comment:
This debug print statement should be removed before merging.
Are you still debugging the issue on Windows?

Reply:
> Are you still debugging the issue on Windows?

No, this was fixed in  #4682.